### ISSUE TITLE
Unable to build. Cannot find module.

### ISSUE THREAD

**Issue by 3DStephen** [ue4plugins/GLTFWebViewer#1]
When running 'npm run start', after waiting for a minute, it fails with the error below.

`Failed to compile`
`./src/icons/Check.svg (./node_modules/@svgr/webpack/lib?-svgo,+titleProp,+ref!./src/icons/Check.svg)
Error: Cannot find module 'D:\Documents\Unreal Projects\GLTFWebViewer-main\GLTFWebViewer\node_modules\@babel\compat-data\data\corejs3-shipped-proposals'`
`This error occurred during the build time and cannot be dismissed.`

**Comment by kevinraz** [comment]
Hey @3DStephen - I fixed this issue with:
`sudo npm update --depth 5 @babel/compat-dat`

**Comment by ArosPrince** [comment]
//edit: OK, finally figured it out. There's a typo in kevinraz's solution. It should be:

`npm update --depth 5 @babel/compat-data`

(missing _a_ at the end)

**Comment by aidinabedi** [comment]
Unfortunately, I can't seem to reproduce this issue. What version of node are you guys running?

And after running the suggested fix, `npm start` stops working for me:
```
Failed to compile.

./node_modules/@pmmmwh/react-refresh-webpack-plugin/src/runtime/BabelDetectComponent.js
Error: [BABEL] D:\dev\gltf-web-viewer\node_modules\@pmmmwh\react-refresh-webpack-plugin\src\runtime\BabelDetectComponent.js: Could not find plugin "proposal-class-static-block". Ensure there is an entry in ./available-plugins.js for it. (While processing: "D:\\dev\\gltf-web-viewer\\node_modules\\babel-preset-react-app\\dependencies.js$0")
    at Array.map (<anonymous>)
    at Generator.next (<anonymous>)
    at Generator.next (<anonymous>)
```

Would love to get to the bottom of this and include a fix for everyone directly in `main`.

**Comment by ArosPrince** [comment]
v14.15.1
(Windows in case it matters)

I tried two different computers, both end up with the same problem.

**Comment by ArosPrince** [comment]
I've been able to reproduce this on multiple Windows computers. It has to be something with the `package-lock.json`.

After I run the aforementioned script package-lock gets modified like this:

![image](https://user-images.githubusercontent.com/25080989/134070201-95739bc0-4ebb-4457-9dfd-2231592c08c8.png)

...and then it works.

If I however then delete node_modules and install again with this modified package-lock, it will end up with different error:

```
Uncaught Error: Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL] c:\Temp\TEST\node_modules\webpack\hot\dev-server.js: Could not find plugin "proposal-class-static-block". Ensure there is an entry in ./available-plugins.js for it. 
```

I am also not able `npm run build` the project:

```
Creating an optimized production build...
Browserslist: caniuse-lite is outdated. Please run the following command: `npx browserslist --update-db`
Failed to compile.

ENOENT: no such file or directory, open '<mypath>\build\index.html'


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @epic/gltf-web-viewer@1.29.1 build: `react-app-rewired build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @epic/gltf-web-viewer@1.29.1 build script.
```

**Comment by aidinabedi** [comment]
Fixed! (I was running node v12.16.3)

The issue seems to be the result of a change first introduced in node 12.17.0. As @ArosPrince noted, updating only `@babel/compat-data` in isolation will have undesirable side effects. So I identified all the affected dependencies and updated them in `package-lock.json`. Please try the latest `main` and if you still experience any issues just let me know.

P.S. The readme's prerequisite node version has also been updated accordingly.
